static/css/components/work.less

```diff
@@ -73,3 +73,8 @@ div.editionTools {
     }
   }
 }
+
+.print-disabled-download {
```

Comment:
Please let's not put this here. This really doesn't belong in this file and makes it impure somehow.

Reply:
print-disabled-download has a specific purpose - to disable downloads when printing I'm guessing? This is not what we're using it for here which is bad (note this also impacts things using openlibrary/macros/daisy.html which might be used outside the works component).

The one I'm seeing on the books page seems to be part of the read panel (static/css/components/read-panel.less). Thus I'd suggest adding a new class panel__download using the BEM notation to tie this to the ReadPanel component it belongs to and moving this style there with the class panel__download instead.

Also this should be above the media query - i don't have a stylelint rule forbidding this yet, but the media queries should be the last things listed.

Reply:
Alternatively consider using the smaller class from static/css/base/helpers-misc.less ?  (and importing it in page-book.css)